src/directives/routeClass.js

```diff
@@ -0,0 +1,28 @@
+class routeClassDirective {
+  constructor(Route) {
+
+    this.restrict = 'A';
+
+    this.link = (scope, iElement, iAttrs) => {
+      scope.$watch(() => {
+        const routeClassDefinition = scope.$eval(iAttrs['routeClass'])
```

Comment:
if there was a Route.isCurrentBinding() this would clean up a bit and would read more explicitly